api-client/src/main/java/com/xing/api/data/profile/LegalInformationComplete.java

```diff
@@ -0,0 +1,43 @@
+package com.xing.api.data.profile;
+
+import com.squareup.moshi.Json;
+
+import java.io.Serializable;
+
+/** Represents full legal information */
+public class LegalInformationComplete implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Json(name = "content")
+    private String content;
+
+    public LegalInformationComplete(String content) {
+        this.content = content;
+    }
+
+    public String content() {
+        return content;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
`if(!(o instanceof LegalInformationComplete))` is better, will not break the equals contract in case of inheritance.